src/java.base/share/classes/java/lang/runtime/SwitchBootstraps.java

```diff
@@ -175,8 +175,8 @@ private static int doTypeSwitch(Object target, int startIndex, Object[] labels)
      * arguments are an array of case labels which must be non-null and of type
```

Comment:
This sentence can still be improved and made cleared. Example:

> The static arguments are used to encode the case labels associated to the `switch` construct, where each label can be encoded as a `String` (e.g. to represent an enum constant), or, alternatively, as a `Class` (e.g. to represent a type test pattern whose type is an enum type).

Reply:
Thanks. I've updated the javadoc and improved the validation a little, checking the `Class` objects provided.